node/src/admin_client.ts

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as grpc from '@grpc/grpc-js';
+
+const empty = require('google-protobuf/google/protobuf/empty_pb.js');
```

Comment:
If the code is typescript and es6, isn't it easier to use `import` instead of the node `require` function which you are already using at the top anyway?

Reply:
Sorry, I didn't know there's a type declaration package for `google-protobuf`.
I added it in https://github.com/scalar-labs/scalar-admin/pull/9/commits/a3dbc0b3794032ee5b20b13ca82745a8b67f0191



I also had a new example to test the unpause function in https://github.com/scalar-labs/scalar-admin/pull/9/commits/d27c06890b5c2488a33b1b4ff59d04995efd8562

Reply:
Sorry, I didn't know there's a type declaration package for `google-protobuf`.
I added it in https://github.com/scalar-labs/scalar-admin/pull/9/commits/a3dbc0b3794032ee5b20b13ca82745a8b67f0191



I also had a new example to test the unpause function in https://github.com/scalar-labs/scalar-admin/pull/9/commits/d27c06890b5c2488a33b1b4ff59d04995efd8562